lib/chef/provider/package/powershell.rb

```diff
@@ -115,7 +118,8 @@ def build_powershell_package_command(command, version = nil)
           command = [command] unless command.is_a?(Array)
           cmdlet_name = command.first
           command.unshift("(")
-          %w{-Force -ForceBootstrap}.each do |arg|
+          # -WarningAction SilentlyContinue is used to suppress the warnings from stdout
+          %w{-Force -ForceBootstrap -WarningAction SilentlyContinue}.each do |arg|
```

Comment:
if we are adding -WarningAction SilentlyContinue here then why in test cases https://github.com/chef/chef/pull/9946/files#diff-4efcbd80710bb0976c44af3650e95528R127 we added that for each command. If I am not wrong and getting it correctly isn't this get appended to those command automatically ?

Reply:
@Vasu1105, I have added the 'WarningAction SilentlyContinue' in specs, because we are compairing the commands that generating from the parrticular method with the commands used in specs.